src/mark.js

```diff
@@ -776,6 +828,20 @@ class Mark { // eslint-disable-line no-unused-vars
      *   <li><i>limiters</i>: A custom array of string limiters for accuracy
      *   "exactly" or "complementary"</li>
      * </ul>
+     * @property {"disable"|"enable"|"includeSpaces"}
```

Comment:
Please outsource the JSDOC as a `@typedef` like [here](https://github.com/julmot/mark.js/blob/master/src/mark.js#L298). This way it's possible to reference it in other functions.